optimize/client/src/components/Reports/ReportView.test.js

```diff
@@ -165,48 +174,32 @@ describe('Download CSV', () => {
   });
 
   it('should show a download csv button even if the result is 0', () => {
-    const node = shallow(<ReportView report={{...report, result: {measures: [{data: 0}]}}} />);
+    const node = shallow(
+      <ReportView report={{...rawDataReport, result: {measures: [{data: 0}]}}} />
+    );
     expect(node.find(DownloadButton)).toExist();
   });
 
   it('should show a download csv button even if the result is null', () => {
-    const node = shallow(<ReportView report={{...report, result: {measures: [{data: null}]}}} />);
+    const node = shallow(
+      <ReportView report={{...rawDataReport, result: {measures: [{data: null}]}}} />
+    );
 
     expect(node.find(DownloadButton)).toExist();
   });
 
   it('should not show a download csv button for multi-measure reports', () => {
     const node = shallow(
-      <ReportView report={{...report, result: {measures: [{data: 0}, {data: 12}]}}} />
+      <ReportView report={{...rawDataReport, result: {measures: [{data: 0}, {data: 12}]}}} />
     );
     expect(node.find(DownloadButton)).not.toExist();
   });
 
-  it('should not show a download csv button if the visualization is number', () => {
-    const node = shallow(
-      <ReportView report={{...report, data: {...report.data, visualization: 'number'}}} />
-    );
+  it('should not show a download csv button if the view is not rawData', () => {
+    const node = shallow(<ReportView report={report} />);
 
     expect(node.find(DownloadButton)).not.toExist();
   });
-
-  it('should calculate total entries correctly for different report types', () => {
-    const node = shallow(
-      <ReportView report={{...report, result: {type: 'number', measures: [{data: 12}]}}} />
-    );
-
-    expect(node.find(DownloadButton).prop('totalCount')).toBe(1);
-
-    node.setProps({report: {...report, result: {type: 'map', measures: [{data: [{}, {}, {}]}]}}});
-
-    expect(node.find(DownloadButton).prop('totalCount')).toBe(3);
-
-    node.setProps({
-      report: {...report, result: {type: 'raw', measures: [{data: [{}]}], instanceCount: 20}},
-    });
-
-    expect(node.find(DownloadButton).prop('totalCount')).toBe(20);
-  });
```

Comment:
Why did we remove this test completely? Can we adapt it for the raw data report?

Reply:
I can do that